SPECS/dracut/dracut.spec

```diff
@@ -156,6 +156,7 @@ ln -srv %{buildroot}%{_bindir}/%{name} %{buildroot}%{_sbindir}/%{name}
 %dir %{dracutlibdir}/modules.d
 %{dracutlibdir}/modules.d/*
 %exclude %{_libdir}/kernel
+%exclude %{_libdir}/dracut/modules.d/20overlayfs/*
```

Comment:
nit: Use `%{dracutlibdir}` instead of ` %{_libdir}/dracut` for the sake of consistency.

Reply:
Fixed, thanks.